demo/movielens-1m-ps/movielens-1m-ps.py

```diff
@@ -0,0 +1,214 @@
+import os, sys
+import tensorflow as tf
+from tensorflow.keras.layers import Dense
+import tensorflow_datasets as tfds
+import tensorflow_recommenders_addons as tfra
+
+tf.compat.v1.disable_v2_behavior()
```

Comment:
`disable_eager_execution` and `disable_resource_variables ` are already called in `disable_v2_behavior `. [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/compat/v2_compat.py#L94-L96)

Reply:
accept